### PR TITLE
Change fan guide to use t2fand and remove 'secondary option' from touchbard

### PR DESCRIPTION
**Pull request by NoaHimesaka1873**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
Here you should add instructions to add the pacman repo as well.

**Comment by NoaHimesaka1873** [comment]
> Here you should add instructions to add the pacman repo as well.

I didn't add it because I assumed the user would have followed t2linux Arch installation guide, which of course adds the necessary repositories. I'll add warning to indicate that.

**Comment by AdityaGarg8** [comment]
I'd rather separate fan into 3 sections

1st for Arch based distros
2nd for Debian and Ubuntu based distros
3rd for rest cases

1st and 2nd having instructions to add repo and install the package.

**Comment by NoaHimesaka1873** [comment]
Damn linter sucks it can't even properly parse numbered list

File: docs/guides/fan.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+
+        1. Clone the repository by running `git clone https://github.com/NoaHimesaka1873/t2fand`
+
+        2. Change directory into the cloned repository.
+
+        3. Install it by running `make install`.
+
+2. Enable daemon by running `sudo systemctl enable --now t2fand`.
+
+3. Edit the config and restart the daemon if needed.
```
Comment (AdityaGarg8) [inline docs/guides/fan.md:33]: I do not find any instructions to restart the daemon. Can you elaborate this?

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/fan.md (hunk 1 above)
Comment (NoaHimesaka1873) [reply docs/guides/fan.md]: Done.

**Pull request merged**